client/info_client.go

```diff
@@ -11,8 +11,6 @@ import (
 // InfoClient collects all Avalanchego info.Client methods common
 // to Rosetta Clients
 type InfoClient interface {
-	GetNetworkName(context.Context, ...rpc.Option) (string, error)
-	GetNetworkID(context.Context, ...rpc.Option) (uint32, error)
```

Comment:
GetNetworName was already not used; dropped for cleanup.
GetNetworkID is dropped in this PR and replaced by methods in avalanchego constants package